modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -148,7 +150,8 @@ const DEFAULT_PROPS: Props = {
   updateTransforms: true,
   viewportTraversersMap: null,
 
-  headers: {},
+  headers: null,
```

Comment:
I recommend we drop the `headers` option and simply use the `loadOptions.fetch`

Reply:
It is breaking for Tiles3DLayer:
https://github.com/visgl/deck.gl/blob/f73ab0550231cd9b3fd6ec7363512fa38d178cd8/modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js#L120

Reply:
if deck needs to be fixed (e.g) to make sure fetch options are merged instead of overwritten then let's make a PR there too.

For the end uses sake I really want us to standardize on the loadOptions: {fetch: {headers: ...}} convention in all places. Deck has been carefully updated to follow this convention.

Reply:
Done